src/main/java/org/wso2/carbon/connector/KafkaConnectConstants.java

```diff
@@ -83,7 +83,8 @@ public class KafkaConnectConstants {
     public static final String PARTITION_NO = "partitionNo";
     public static final String PARAM_KEY = "key";
 
-    public static final String METHOD_NAME = "publishMessages:";
+    public static final String KAFKA_HEADER_PREFIX = "kafka.kafkaHeaderPrefix";
+    public static final String DEFAULT_KAFKA_HEADER_PREFIX = "kafkaHeader_";
```

Comment:
Shouldn't the default be "publishMessages:" to preserve backward compatibility?

Reply:
The reason behind making this prefix configurable was to make the both Kafka connector and Kafka Inbound Endpoint consistent from their respective next release onwards. Hence, wanted to default the prefix to a meaningful name that can be used across the connector and inbound endpoint. Also, using a new connector version means, re-doing the configurations. So, one can configure the property as below to preserve backward compatibility in their integration use cases. WDYT of that?

`<kafkaHeaderPrefix>publishMessages:</kafkaHeaderPrefix>`

Reply:
+1. Fixed with https://github.com/wso2-extensions/esb-connector-kafka/pull/78/commits/4159e818bf870eda9e6cbb54c9f61520e78f58e2